docs/tracing.md

```diff
@@ -89,6 +89,25 @@ Containers agent:
   The [Jaeger "all-in-one" Docker image][jaeger-all-in-one] method
   is the quickest and simplest way to run the collector for testing.
 
+  - It is worth noting that the **default collector endpoint** URL 
+    is set to http://localhost:14268/api/traces
+    When custom endpoint URL is used, make sure you are setting the correct
+    **collector** port.
+
+  - Following is a example command to run all Jaeger parts in a docker
```

Comment:
We previously did not include the command and instead directed users to Jaeger's instructions since this command has changed over time and theirs is the most current. I do not mind keeping it in though.

Reply:
I agree with @cmaf , this could be a link to jaeger's instruction. cc @YushuoEdge